spec/features/incoming_mail_handler_spec.rb

```diff
@@ -0,0 +1,79 @@
+describe "Handles incoming email" do
+  let(:proposal) { FactoryGirl.create(:proposal, :with_parallel_approvers) }
+  let(:approval) { proposal.individual_approvals.first }
+  let(:mail) { CommunicartMailer.actions_for_approver(approval) }
+  let(:mandrill_inbound_noapp) { File.read(RSpec.configuration.fixture_path + '/mandrill_inbound_noapp.json') }
+
+  def mandrill_payload_from_message(mail_msg)
+    headers = {}
+    mail_msg.header.fields.each do |header|
+      headers[header.name] = header.value
+    end
+    msg = {
+      'subject'  => mail_msg.subject,
+      'template' => nil,
+      'tags'     => [],
+      'from_email' => mail_msg.to[0], # NOTE this is switched with 'email' because mail_msg is what we are *sending*
+      'email'      => mail_msg.from[0],
+      'sender'     => nil,
+      'text'       => mail_msg.text_part.body.encoded,
+      'html'       => mail_msg.html_part.body.encoded,
+      'raw_msg'    => mail_msg.to_s,
+      'headers'    => headers,
+    }
+    [ { 'event' => 'inbound', 'msg' => msg } ]
+  end
+
+  with_env_vars(NOTIFICATION_FALLBACK_EMAIL: 'nowhere@some.gov', NOTIFICATION_FROM_EMAIL: 'noreply@some.gov') do
+    it "should forward non-app email to NOTIFICATION_FALLBACK_EMAIL" do
```

Comment:
this is more of a general thought and not necessarily something to take action on now: what about removing "should" from our spec descriptions:

> Finally, don’t begin examples names with the word ‘should’.  It’s redundant and results in hard to read spec output. 

> Write examples by starting with a present tense verb that describes the behavior.

http://blog.carbonfive.com/2010/10/21/rspec-best-practices/